app/lib/ability.rb

```diff
@@ -14,10 +14,13 @@ def initialize(user)
       # All admins who are also state file
       can :manage, :all
       unless user.state_file_admin?
-        cannot :manage, StateFileAzIntake
-        cannot :manage, StateFileNyIntake
+        StateFile::StateInformationService.intake_classes.each do |intake_class|
+          cannot :manage, intake_class
+        end
+        # Enumerate classes here...
         cannot :manage, StateFile1099G
         cannot :manage, StateFileDependent
+        cannot :manage, StateFileW2
```

Comment:
I added this because I noticed that we had missed it.

Reply:
ooo good catch